assets/app/features/upgrade/templates/upgrade7/database-configuration.jade

```diff
@@ -0,0 +1,32 @@
+h2(translate='') upgrade7.steps.database-configuration.title
+
+uib-tabset(active='active')
+  uib-tab(index='0', heading='Create a Database')
+    form
+      .form-group
+        label(translate='') upgrade7.steps.database-configuration.form.username
+        input.form-control(type='text', required='', ng-model='username')
+      .form-group
+        label(translate='') upgrade7.steps.database-configuration.form.password
+        input.form-control(type='text', required='', ng-model='model.password')
```

Comment:
Good try to use ngModels, @bisht-richa! :)
Now we would need to create a new model in the controller's view model to map and handle those models. Something like
````
vm.form = {
    username: '',
    password: '',
    server: '',
    port: '',
    table_prefix: ''
}
```

Then in this view, we'd just bind them as `ng-model='vm.form.server'`

Please, let me know if you need help with it.